### ISSUE TITLE
Cannot use comments to instruct Istanbul to ignore lines of code

### ISSUE THREAD

**Issue by ehmicky** [duereg/gulp-coffee-istanbul#4]
It is not possible to use comments to instruct Istanbul to ignore lines of code. In JavaScript:

    /* istanbul ignore next */

In CoffeeScript I tried without success:

    ### istanbul ignore next ###
    # istanbul ignore next
    # /* istanbul ignore next */
    ### /* istanbul ignore next */ ###

**Comment by duereg** [comment]
Yeah, this makes sense. The code to instrument coffee is custom - I'll have to look into how the JS version works and see what I can do to fix it. 

Thanks for the heads up.

**Comment by fabiancook** [comment]
+1

**Comment by Zolmeister** [comment]
Here is the fix. https://github.com/Constellation/ibrik/pull/32

**Comment by duereg** [comment]
great! I'll merge this change as soon as it's released by ibrik.

**Comment by mkjois** [comment]
This issue is still relevant. I can't seem to ignore CS lines with any type of comments.

**Comment by duereg** [comment]
@Zolmeister, @mkjois I *think* this is resolved. Let me know if 0.9.0 resolves your issues.
